sphinxcontrib/datatemplates/directive.py

```diff
@@ -136,6 +138,22 @@ class DataTemplateWithEncoding(DataTemplateBase):
                               })
 
 
+class DataTemplateNothing(DataTemplateBase):
+    """
+    .. rst:directive:: .. datatemplate:nothing:: source-path
```

Comment:
If source-path isn't used, we shouldn't include it in the syntax.

Reply:
Actually source-path is available to the template as source (same for all directives) but it should be made clear.

Reply:
Removed. We should put something about argument and option handling in the docs though.